File: src/ServiceControl/Recoverability/Grouping/Retries/RetryType.cs
namespace ServiceControl.Recoverability
{
    public enum RetryType
    {
        Unknown = 0,
        SingleMessage = 1,
        FailureGroup = 2,
        MultipleMessages = 3,
        AllForEndpoint = 4,
        All = 5, 
        ByQueueAddress = 6
    }
}

File: src/resources/RetryType.ts
export enum RetryType {
  Unknown = "Unknown",
  SingleMessage = "SingleMessage",
  FailureGroup = "FailureGroup",
  MultipleMessages = "MultipleMessages",
  AllForEndpoint = "AllForEndpoint",
  All = "All",
  ByQueueAddress = "ByQueueAddress",
}
